src/components/implementation/no_interface/llbooter/llbooter.c

```diff
@@ -282,6 +283,8 @@ comps_init(void)
 		assert(comp);
 		elf_hdr = (void *)args_get(imgpath);
 
+		prot_domain_t pd = protdom_ns_asid_alloc(ns_asid);
```

Comment:
Note to self to check why this function takes in the whole array, and how it sanely handles that.